test/end-to-end.js

```diff
@@ -1043,6 +1043,63 @@ describe('end-to-end', function() {
         .end(done);
     }
   });
+
+  describe('Multi-project workspace support', function() {
+    beforeEach(givenEmptySandbox);
+    afterEach(function setWorkspaceToSandboxDir() {
+      process.env.WORKSPACE_DIR = SANDBOX;
+    });
```

Comment:
afterEach(setWorkspaceToSandboxDir);

Also subjective is beforeEach can be just "before" and afterEach can just be "after" since you only one test. The next person adding a test can then decide whether they want it for their test without having to modify the "Each" part (and add it if they deem it necessary).